docs/guide/recovery.md

```diff
@@ -20,12 +20,12 @@ nav_order: 2
   <img loading="lazy" src="/assets/images/twrp.png" width="30%"/>
 </p>
 
-Installing custom ROM's via recovery has been the standard since Android early times, and today it's still the way to go in most devices.
+Installing custom ROMs via a recovery has been the standard since Android early times, and today it's still the way to go in most devices.
 This guide will not go in deep details about how to install a custom recovery, as the procedure might differ from device to device, so please refer to the XDA community page of [your device]({% link devices/index.md %}).
 
-To install UN1CA, a custom recovery and/or other unofficial images, your device's bootloader **must be unlocked**, if you're unsure please give a look at the ["Unlocking the bootloader"]({% link guide/bootloader.md %}) page.
+To install UN1CA, a custom recovery and/or other unofficial images are required and your device's bootloader **must be unlocked**. If you are unsure please take a look at the ["Unlocking the bootloader"]({% link guide/bootloader.md %}) page.
```

Comment:
Remove "are required and" as that's not what I meant

Reply:
This also must match the Download mode guide:
https://github.com/BlackMesa123/UN1CA/blob/073aed5b12625bd6e5550f4f75a107fd19423fe2/docs/guide/download.md?plain=1#L30

Reply:
Fixed.